common/client-libs/validator-client/src/nyxd/mod.rs

```diff
@@ -158,12 +158,13 @@ impl NyxdClient<HttpClient> {
     }
 }
 
-impl NyxdClient<ReqwestRpcClient> {
+#[allow(deprecated)]
```

Comment:
I guess you know what you're doing but why the deprecated `fn` use? Are there no replacements?